more-dom-manipulation/app.js

```diff
@@ -0,0 +1,83 @@
+//1. Change the font:
+const pageFont = document.querySelector(".page");
+pageFont.style.fontFamily = "Arial, sans-serif";
+
+//2.Replace each of the span tags
+//NickName:
+const sTags = document.querySelector("span#nickname");
+sTags.replaceWith("Eman");
+
+//Favorite Animal:
+const sTags2 = document.querySelector("span#favorite");
+sTags2.replaceWith("Cats");
+
+//Hometown:
+const sTags3 = document.querySelector("span#hometown");
+sTags3.replaceWith("Abha");
+
+//3. Give each li the class of "listitem"
+const newListItems =document.createElement("li.listitem")
+
+
+
+
+//4. Change each li's text color to "rebeccapurple"
+const listItem = document.querySelector("ul");
+listItem.style.color ="rebeccapurple";
+
+
+//5. Create a new img element and set its src attribute to a picture of you. Put that new img element in the div with the class of "profilePicture"
+let newImg = document.createElement("img");
+newImg.setAttribute("src", "https://htdraw.com/wp-content/uploads/2018/04/How-to-draw-Cute-Girl-step-by-step-easy-Draw-so-cute-300x281.png");
+document.body.appendChild(newImg);
+newImg=document.createElement("div.profilePicture");
```

Comment:
the div.profilePicture already exists, what you want is select it using `querySelector` then append the newImg to it